rancid-tomatillos/src/containers/Login/Login.js

```diff
@@ -29,17 +30,16 @@ class Login extends Component {
       body: JSON.stringify(this.state)
     }
 
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/login', options)
-      .then(res => {
-        if(!res.ok) {
-         throw Error('Incorrect Username/Password')
-        }
-        return res.json()
-      })
+    fetchUser(options)
       .then(data => {
-        console.log(data)
-        this.props.addUser(data)
+        this.props.addUser(data.user.id)
         this.props.updateLoggedIn(this.props.isLoggedIn)
+        return data.user.id
+      })
+      .then(id => {
+        fetchRatings(id)
+          .then(data => this.props.updateUserRatings(data.ratings))
+          .catch(err => console.log(err))
```

Comment:
Should we have an error property in our redux store to update on all of our catches??